static-site/content/blog/2024-06-18-h5n1-cattle-outbreak-analysis-and-resources.md

```diff
@@ -0,0 +1,132 @@
+---
+author: "Trevor Bedford, Louise Moncla, Jover Lee, James Hadfield, John Huddleston"
```

Comment:
I know it's only a blog post and maybe this doesn't matter, but I feel strange about being listed last author here since I did the least amount of work. Louise seems a better fit for the "last author".

Reply:
Thanks John! Resolved in 3901038276d3f1bbe964bcb3a0652ac54d373e38.